CI integration in revision grid adjustments

pmiossec opened the pull request:
## Proposed changes

- Update AppVeyor display string to display the more important information first
- Update AzureDevops display string to display the more important information first
- Increase the width of the "Build status text" column (because we can't resize it)


## Screenshots <!-- Remove this section if PR does not change UI -->

AppVeyor:
### Before

![image](https://user-images.githubusercontent.com/460196/51251085-b7bf4600-1998-11e9-9446-0590438ca278.png)

### After

![image](https://user-images.githubusercontent.com/460196/51251118-cefe3380-1998-11e9-95ae-99ca9d388f3b.png)

AzureDevops:
### Before

![image](https://user-images.githubusercontent.com/460196/51251235-243a4500-1999-11e9-9820-ede7db4039d7.png)

### After

![image](https://user-images.githubusercontent.com/460196/51251157-f0f7b600-1998-11e9-9448-0d679b98d153.png)


## Test methodology <!-- How did you ensure quality? -->

- Running the application (see screenshots)


## Test environment(s) <!-- Remove any that don't apply -->

- GIT 2.20
- Windows 10

<!-- Mention language, UI scaling, or anything else that might be relevant -->

gerhardol commented on Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs:
```diff
@@ -523,7 +523,7 @@ public void ChangeProgressCounter()
         public void UpdateDescription()
         {
-            Description = Id + " " + DisplayStatus + " " + _buildDurationFormatter.Format(Duration) + TestsResultText + PullRequestText;
+            Description = DisplayStatus + " " + _buildDurationFormatter.Format(Duration) + TestsResultText + PullRequestText + " " + Id;
```
The Display status is already in the color, why have that first?

RussKie commented:
Very good question.
Philippe do you feel the color indicator not sufficiently clear to require
a text (like 'success', 'fail').
If not, can we drop the text?


On Fri, Jan 18, 2019, 9:59 AM Gerhard Olsson <notifications@github.com
wrote:

> *@gerhardol* commented on this pull request.
> ------------------------------
>
> In Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs
> <https://github.com/gitextensions/gitextensions/pull/6116#discussion_r248873496>
> :
>
> > @@ -523,7 +523,7 @@ public void ChangeProgressCounter()
>
>          public void UpdateDescription()
>          {
> -            Description = Id + " " + DisplayStatus + " " + _buildDurationFormatter.Format(Duration) + TestsResultText + PullRequestText;
> +            Description = DisplayStatus + " " + _buildDurationFormatter.Format(Duration) + TestsResultText + PullRequestText + " " + Id;
>
> The Display status is already in the color, why have that first?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/gitextensions/pull/6116#pullrequestreview-193882936>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEMyXmsp2ac-1nJ58ujVboouYJSXDpeMks5vEQBFgaJpZM4aDDVw>
> .
>

pmiossec commented:
@RussKie @gerhardol 

>The Display status is already in the color, why have that first?
>Philippe do you feel the color indicator not sufficiently clear to require

The idea behind this PR was to put first the data that bring the more sense (and put last the build number that I don't really find useful.)

I just put the already existing info at the beginning and kept the "build state" first to make things aligned, but I completely agree with you and find it useless (colors are enough). I didn't want to impose the remove of this information to others...

I will remove it.

pmiossec commented:
I have:
* Removed the Status of the build (Success, In Progress, ...)  because color is enough
* Added a tooltip property to display a different information (by rows, more readable)
* AppVeyor: display the PR title in the tooltip

What it looks like now:

![image](https://user-images.githubusercontent.com/460196/51543566-22232b00-1e5e-11e9-80b5-cbfd7217ebae.png)

RussKie commented:
@pmiossec could you please rebase on top of latest master and add your sign off under [Git Extensions Project Contributors Certification of Origin and Rights](https://github.com/gitextensions/gitextensions/blob/master/contributors.txt) as part of this PR.
Thank you.

gerhardol commented on Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs:
```diff
@@ -538,5 +547,5 @@ private string DisplayStatus
             }
         }
-    }
+        }
```
whitespace

RussKie commented:
Go for it, we can always tweak it again later
Thank you

On Fri, 25 Jan 2019 at 07:44, Gerhard Olsson <notifications@github.com>
wrote:

> *@gerhardol* approved this pull request.
>
> Will merge unless RussKie protests or or does it first
> (ignoring the minor whitespace and that appveyor is not as I prefer)
> ------------------------------
>
> In Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs
> <https://github.com/gitextensions/gitextensions/pull/6116#discussion_r250768098>
> :
>
> > @@ -538,5 +547,5 @@ private string DisplayStatus
>                  return "In progress" + new string('.', _buildProgressCount) + new string(' ', 3 - _buildProgressCount);
>              }
>          }
> -    }
> +        }
>
> whitespace
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/gitextensions/pull/6116#pullrequestreview-196238316>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEMyXmlX0rhvCxx1DA7Zl4eNjaQAHXWOks5vGhsugaJpZM4aDDVw>
> .
>

gerhardol commented:
@pmiossec 
Can you please rebase, dropping my commit?
The change to contributors.txt should be done by you (since I do not rebase, it is, but we will get two merge commits here).

Let us hope that this can be merged before any other changes are merged...

vbjay commented:
This is why I pointed out the cla bot on gutte

On Thu, Jan 24, 2019, 6:41 PM Gerhard Olsson <notifications@github.com>
wrote:

> @pmiossec <https://github.com/pmiossec>
> Can you please rebase, dropping my commit?
> The change to contributors.txt should be done by you (since I do not
> rebase, it is, but we will get two merge commits here).
>
> Let us hope that this can be merged before any other changes are merged...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/gitextensions/pull/6116#issuecomment-457400384>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADdhsWFVfTIo1cTRYTPgCTRlSD-Fb0Vhks5vGkSRgaJpZM4aDDVw>
> .
>

RussKie commented:
unfortunately we'll have a bit contention and race conditions on
contributors.txt until things stabilise...
It may be easier to send a PR with DCO separately.

On Fri, Jan 25, 2019, 10:42 AM Jay Asbury <notifications@github.com wrote:

> This is why I pointed out the cla bot on gutte
>
> On Thu, Jan 24, 2019, 6:41 PM Gerhard Olsson <notifications@github.com>
> wrote:
>
> > @pmiossec <https://github.com/pmiossec>
> > Can you please rebase, dropping my commit?
> > The change to contributors.txt should be done by you (since I do not
> > rebase, it is, but we will get two merge commits here).
> >
> > Let us hope that this can be merged before any other changes are
> merged...
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/gitextensions/gitextensions/pull/6116#issuecomment-457400384
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/ADdhsWFVfTIo1cTRYTPgCTRlSD-Fb0Vhks5vGkSRgaJpZM4aDDVw
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/gitextensions/pull/6116#issuecomment-457400666>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEMyXhunIGCyOQsCsNn7WmWvcC8NWa7Uks5vGkTngaJpZM4aDDVw>
> .
>
